src/context/videoLikeContext.js

```diff
@@ -21,14 +35,15 @@ const LikeProvider = ({children}) => {
         }
       });
       console.log(response)
+      // setLikedVideo(prev=>prev.filter(item=> item._id !==response.data.likes[0]._id ))
     }
```

Comment:
remove unused code

Reply:
I have to made many changes so I will do it later